website/architecture/bundled-hermes.md

```diff
@@ -83,7 +83,9 @@ Users building React Native App, with the New Architecture, on Windows machines
 
 * Make sure the [environment is configured properly](https://reactnative.dev/docs/environment-setup), with Android SDK & node.
 * Install [cmake](https://community.chocolatey.org/packages/cmake) with Chocolatey
-* Install [Visual Studio 22 Community edition](https://visualstudio.microsoft.com/vs/community/) - Picking only the C++ desktop development is sufficient.
+* Install either:
+  * [Visual Studio 22 Community edition](https://visualstudio.microsoft.com/vs/community/) - Picking only the C++ desktop development is sufficient.
+  * [Build Tools for Visual Studio 2022](https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2022).
```

Comment:
```suggestion
  * [Build Tools for Visual Studio 2022](https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2022).
  * [Visual Studio 22 Community Edition](https://visualstudio.microsoft.com/vs/community/) - Picking only the C++ desktop development is sufficient.
```

Small naming nit, plus suggestion to move Build Tools above, since they are a little bit lighter installation footprint than even almost bare VS22.